tools/runners/run-unittests.sh

```diff
@@ -97,7 +104,11 @@ rm -f $UNITTEST_TEMP
 
 ratio=$(echo $passed*100/$total | bc)
 
-echo "[summary] ${DIR#$ROOT_DIR}/unit-*: $passed PASS, $failed FAIL, $total total, $ratio% success"
+echo -e "\n [summary] ${DIR#$ROOT_DIR}/unit-*\n"
+echo -e " TOTAL: $total"
+echo -e "$TERM_GREEN PASS: $passed"
+echo -e "$TERM_RED FAIL: $failed $TERM_EMPTY\n"
+echo -e " Success: $ratio%\n"
```

Comment:
And why are all the diagnostics indented by _one_ space? It might be just me but that's very strange to my eyes.